lib/pbench/test/unit/server/conftest.py

```diff
@@ -323,6 +323,49 @@ def attach_dataset(create_drb_user, create_user):
         ).add()
 
 
+@pytest.fixture()
+def more_datasets(
+    client, server_config, create_drb_user, create_admin_user, attach_dataset
+):
+    """
+    Supplement the conftest.py "attach_dataset" fixture with a few more
+    datasets so we can practice various queries. In combination with
+    attach_dataset, the resulting datasets are:
+
+        Owner   Access  Date        Name
+        ------- ------- ----------- ---------
+        drb     private 2020-02-15  drb
+        test    private 2002-05-16  test
+        drb     public  2020-02-15  fio_1
+        test    public  2002-05-16  fio_2
```

Comment:
How does this result in 4 things when there are 2 `Dataset().add()` calls below?

Reply:
Yes, I'd considered breaking it up by fixture, but having the list in one place is more useful. Maybe I should have put a *subset* of this list in the `attach_dataset` fixture, however, to complete the set. (I even thought of just collapsing them into one, but I didn't want to bother dealing with any fallout from that if it affects "older" test cases.)